app/views/spam2/_revisions.html.erb

```diff
@@ -0,0 +1,68 @@
+<script>
+$(document).ready(function() {
+  $("#revision_table").DataTable({
+      "order": [[ 1, "desc" ]],
+      stateSave: true,
+      "lengthMenu": [[30, -1, 0], ["Auto", "All", "None"]],
+       "autoWidth": false,
+       "search": {
+          "regex": true
+        },
+        "info": false,
+        "scrollX": true,
+    });
+    $('#selectall').click(function () {
+        $('.selectedId').prop('checked', this.checked);
+    });
+    $('.selectedId').change(function () {
+        var check = ($('.selectedId').filter(":checked").length == $('.selectedId').length);
+        $('#selectall').prop("checked", check);
+    });
+    });
+</script>
+
+<div class="card" id="table-card">
```

Comment:
This Ruby block is also very similar to the one from `app/views/spam2/_comments.html.erb`. Refactoring this wouldn't hurt. @Uzay-G, what do you think?

Reply:
I added it as function in spam2.js